### ISSUE TITLE
Feature request in insights and cards

### ISSUE THREAD

**Issue by mruiter** [robertraaijmakers/com.iungo#8]
The daily used values.

Would be great if you could also implement the daily used values.

So when you look at the card you see how much water has been used from 00.00 to the time of looking.
Same for solar generated and used from grid.
The last also give ,when overlapping the values on insights , a great view on how green the day was.

**Comment by robertraaijmakers** [comment]
Good idea and have thought about this for a while. But in my opinion is the Iungo the energy manager. So Iungo should be the one implementing this feature. If they do, I will gladly import their API values in Homey (they currently don't have those values available through their API, only in their own app :().

This is to prevent values from getting lost when Homey reboots, or when Iungo is down to get inaccurate values etc.

Another option if Homey themselves would implement some nice daily graphs (with calculations) via Insights. But I think we will have to wait for that for some time.

**Comment by robertraaijmakers** [comment]
Ik heb het verzoek ingediend bij Iungo: https://www.iungo.nl/nl/forum/gewenste-functionaliteiten/132-dagverbruik-uitleesbaar-via-api

**Comment by mruiter** [comment]
Dont know how much you check the forums but its already supported in iungo api 🤘
Maar dan wel met een sommetje 


De API call die hiervoor gebruikt kan worden is datalog_get_day_start_values.


Van alle cumulatieve properties worden dagstartstanden opgeslagen en deze zijn via bovenstaande call op te vragen (per property).

Om van een Object alle property startstanden te krijgen kan je eventueel gebruik maken van:
datalog_get_object_day_start_values (met object ID, starttijdstip en eindtijdstip ).


 
![img_0308](https://user-images.githubusercontent.com/17739535/30236964-3bef32d4-9527-11e7-98be-68ee99d3f7af.PNG)



In het geval van de slimme meter krijg je dan T1, T2, -T1(nT1), -T2(nT2) terug in tijdstempel en waarde paar.
Om het gebruik in kWh van 1 dag te berekenen is de som dan
(T1+T2-nT1-nT2) startvandevolgendedag - (T1+T2-nT1-nT2) startvandedag

Het gebruik van vandaag kan berekend worden door de daystart van vandaag af te trekken van de huidige stand.

Een dag start stand in iungo wordt gelogd als de eerste nieuwe waarde na 00:00 (in de ingestelde tijdzone)

**Comment by robertraaijmakers** [comment]
Daily values are now available through the new energy insights of Homey.
